pages/_app.js

```diff
@@ -40,6 +48,7 @@ export default function App({ Component, pageProps }) {
           plants={plants}
           preferences={preferences}
           handleAddPreference={handleAddPreference}
+          handleEditPreference={handleEditPreference}
```

Comment:
this prop should be 
`onEditPreference={handleEditPreference}`
by common naming convention.